src/backends/plotly.jl

```diff
@@ -900,23 +900,49 @@ function plotly_series_segments(series::Series, plotattributes_base::KW, x, y, z
             mcolor = rgba_string(
                 plot_color(get_markercolor(series, clims, i), get_markeralpha(series, i)),
             )
+            mcolor_next =
+                (i + 1 <= length(series[:markercolor])) ?
+                rgba_string(
+                    plot_color(
+                        get_markercolor(series, clims, i + 1),
+                        get_markeralpha(series, i + 1),
+                    ),
+                ) : mcolor
```

Comment:
```suggestion
            mcolor_next = if i + 1 <= length(series[:markercolor])
                plot_color(
                    get_markercolor(series, clims, i + 1),
                    get_markeralpha(series, i + 1),
                ) |> rgba_string
            else
                mcolor
            end
```

I think this is more readable.